lib/core/viewer/components/pdf-viewer.component.ts

```diff
@@ -166,7 +166,11 @@ export class PdfViewerComponent implements OnChanges, OnDestroy {
     executePdf(pdfOptions: PDFSource) {
         pdfjsLib.GlobalWorkerOptions.workerSrc = 'pdf.worker.min.js';
 
-        this.loadingTask = pdfjsLib.getDocument(pdfOptions);
+        this.loadingTask = pdfjsLib.getDocument({
+            ...pdfOptions,
+            disableAutoFetch: true,
```

Comment:
Should they be in the pdf options?

Reply:
`pdfOptions` config is provided separately for `urlFile` and `blobFile`.  I just thought here  will be better, but  I can make these 2 additions default params and  spread them in `pdfOptions`